Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SavePlot1D.py

```diff
@@ -104,6 +108,15 @@ def getData(self, ws, wkspIndex, label=''):
 
         return (x, y, label)
 
+    def showSpectrum(self, ws, wkspIndex):
+        spectraNum = ws.getSpectrum(wkspIndex).getSpectrumNo()
+
+        # user not specifying which spectra means show them all
+        if len(self.visibleSpectra) <= 0:
```

Comment:
No, but the return value of `len` is a signed integer so I'd rather be paranoid.